### ISSUE TITLE
Error: path is on mount 'C:', start on mount 'D:'

### ISSUE THREAD

**Issue by RemiDav** [sphinx-doc/sphinx#4708]
### Problem
In Spyder, Sphinx crashes when using modules on both C: and D: drive (Windows).

```
The following error occured when calling Sphinx 1.7.1.
Incompatible Sphinx version or doc string decoding failed.
Error message:
path is on mount 'C:', start on mount 'D:'
```

#### Procedure to reproduce the problem
Just load modules from C: and D:.
Working directory in D:.
Have the auto-help loading in spyder activated (loads help when writing a left parenthesis)
Write code calling a function, example: range()

#### Expected results
help loading normally

### Environment info
Sphinx 1.7.1
Spyder 3.2.7
Python 3.6
Windows 10

**Comment by RemiDav** [comment]
Note: Reinstalling completely anaconda solved it. I am afraid to update now.
Anyone reproduced it ?

**Comment by tk0miya** [comment]
The message is not Sphinx's. I guess the error is came from spyder.

**Comment by tk0miya** [comment]
Any updates? If none, I'll close this soon.

**Comment by sein-tao** [comment]
The same problem occurred here.

**Error message:**
```
The following error occured when calling Sphinx 1.7.2.
Incompatible Sphinx version or doc string decoding failed.
Error message:
path is on mount 'C:', start on mount 'E:'
```
**Reproduce of the problem:**
install spyder on `C:`
start spyder with `HOME` on `E:`
in the help pane, type something for help, eg. `str`

**software versions**
spyder  3.2.8
python 3.6.4
Sphinx 1.7.2
windows 10 version 1709

**environment variable**
`HOME=E:\Work`

**Comment by tk0miya** [comment]
I found this is a bug of Sphinx. And I fixed this by #4807 finally.
Thank you for reporting!
